src/System.Diagnostics.Process/tests/ProcessTests.netcoreapp.cs

```diff
@@ -40,5 +40,30 @@ public void Start_StandardInputEncodingPropagatesToStreamWriter()
 
             Assert.Same(encoding, process.StandardInput.Encoding);
         }
+
+        [Fact]
+        public void StartProcessWithArgumentList()
+        {
+            List<string> argList = new List<string>();
+            argList.Add("arg1");
+            argList.Add("arg2");
+
+            ProcessStartInfo psi = new ProcessStartInfo(GetCurrentProcessName(), argList);
+            Process testProcess = CreateProcess();
+            testProcess.StartInfo = psi;
+
+            try
+            {
+                testProcess.Start();
+                Assert.Equal("arg1 arg2", testProcess.StartInfo.Arguments);
+            }
+            finally
+            {
+                if (!testProcess.HasExited)
+                    testProcess.Kill();
+
+                Assert.True(testProcess.WaitForExit(WaitInMS));
+            }
+        }
```

Comment:
We should also have tests that verify:
- behavior when both Arguments and ArgumentList are used
- using the same ProcessStartInfo with arguments to start multiple processes
- if we still have the ctor, behavior when null is passed in, or if we add a setter, behavior when it's set to null